bip-0155.mediawiki

```diff
@@ -193,6 +193,13 @@ Cjdns addresses are simply IPv6 addresses in the <code>fc00::/8</code> range<ref
 
 Yggdrasil addresses are simply IPv6 addresses in the <code>0200::/7</code> range<ref>[https://yggdrasil-network.github.io/faq.html#will-yggdrasil-conflict-with-my-network-routing Yggdrasil FAQ]</ref>. They MUST be sent with the <code>YGGDRASIL</code> network ID.
 
+== Changelog ==
+
+* 0.2.0 (2025-10-01):
+** Add note that Tor v2 is no longer operational.
+* 0.1.0 (2019-02-27):
+** Initial version
```

Comment:
Actually, per BIP3 which specifies the Changelog section, the initial version should be 1.0.0. And this may well be considered a breaking change of the specification?

Reply:
Yes, 1.0.0 and 2.0.0 SGTM. @brunoerg mind updating?